### DISCUSSION TITLE
Query help

### DISCUSSION THREAD

**Discussion by theswapmeat** [supabase/supabase#6164] [General]
Here's the setup of my rather simple db:

countries
{
id as primarykey,
name
}

matches
{
id as primarykey,
hometeamid foreignkey to countries.id,
awayteamid foreignkey to countries.id,
matchdate,
hometeamscore,
awayteamscore
}

picks
{
id as primarykey,
userid as uuid,
hometeamscore,
awayteamscore,
matchid foreignkey to matches.id
}

Querying the matches table to get the hometeamname and awayteam name from the countries table was simple enough:
const response = supabase
    .from("matches")
    .select("*, hometeamname:hometeamid(name), awayteamname:awayteamid(name)")
    .order("id", { ascending: true })
    .order("matchdate", { ascending: true });
  return response;

However, I am getting stuck with how to query the picks table where I want to return the picks.hometeamscore, picks.awayteamscore, hometeamname and awayteamname.  The last two coming from matches and then the countries tables.

**Comment by GaryAustin1** [comment]
If you look thru this section of PostgREST, it might cover your case:
https://postgrest.org/en/stable/api.html#resource-embedding
